src/services/admin.js

```diff
@@ -59,8 +59,6 @@ const getStudentUsers = async (req, res, next) =>
         let users = await User.find({
             "isAdmin": false
         }, {
-            languagesToTeach: 0,
-            languagesToLearn: 0,
             __v: 0,
             matches: 0
```

Comment:
@EAkkanen shall we whether it's only matches 0 students are returning after this? 
This might be something that has been done by a previous team without a proper business knowledge.
Maybe we can ask from clients if it is the case, and fix it accordingly...

Reply:
Huh, I didn't pay attention to that. We can ask the clients but this might be something they're not aware of. I don't know why this part is set up like this.
Do you want me to change something about this request? Leave them commented instead of removing them for example.

Reply:
Nope, I think we can raise this concern at our client meeting... If they say something let's remove it... 
Anyway, just play with some matches and please be confident that "matched" students are not there in this result. If so, let's tell them...
Thanks...
Apart from that, this is good to merge from my end...

Reply:
Seems okay..so if its not what the client requires then we can create another branch for it to make the changes